contracts/Splitter.sol

```diff
@@ -18,10 +25,13 @@ contract Splitter {
         return balance;
     }
 
+    // Comment:  Not sure if that is needed
```

Comment:
To be honest wasn't sure what was the indent of the contract when I started working on it, the problem that solves is too trivial and I tried to make some dummy use case for it, just to make sense to exist. So I choose to make participants initialisation(deploy) time required and after that every participant to be able to check if they are in the scheme.